packages/react/src/primitives/SwitchField/useSwitch.ts

```diff
@@ -1,6 +1,13 @@
 import { ChangeEvent, useCallback, useState } from 'react';
 
-export const useSwitch = (props) => {
+interface UseSwitch {
```

Comment:
Prefer appending `Params` or `Props` to hook param interfaces, for example `UseSwitchParams`/`UseSwitchProps` and using the capitalized hook name for the return type